pkg/kubelet/apis/config/helpers_test.go

```diff
@@ -251,6 +251,8 @@ var (
 		"SeccompDefault",
 		"SerializeImagePulls",
 		"ShowHiddenMetricsForVersion",
+		"ShutdownGracePeriodByPodPriority[*].Priority",
+		"ShutdownGracePeriodByPodPriority[*].ShutdownGracePeriodSeconds",
```

Comment:
This may need to be updated after https://github.com/kubernetes/kubernetes/pull/105685 merges